docs/setup.md

```diff
@@ -212,19 +212,22 @@ To build Linux, Mac, and Windows wheels using GitHub Actions, create a `.github/
             os: [ubuntu-20.04, windows-2019, macos-10.15]
 
         steps:
-          - uses: actions/checkout@v2
+          - uses: actions/checkout@v3
 
           # Used to host cibuildwheel
-          - uses: actions/setup-python@v2
+          - uses: actions/setup-python@v4
+            with:
+              python-version: 3.x
```

Comment:
```suggestion
          - uses: actions/setup-python@v3
```

Reply:
Fixed that and many more files with [search and replace](https://github.com/facebookincubator/fastmod).